docs/pguidance/agent-k8s-install-helm.md

```diff
@@ -0,0 +1,117 @@
+# Install Using Helm
+
+Use the Helm package manager to install the Smart Agent to Kubernetes
+environments.
+
+If you don't have Helm installed already, use kubectl
+to install the Smart Agent. To learn more, see
+[Install the Smart Agent using kubectl](agent-k8s-install-kubectl.md).
+
+## Prerequisites
+
+* Linux kernel version 2.6.32 or higher
+* cap_dac_read_search and cap_sys_ptrace capabilities
+* Terminal or a similar command-line interface application
+* Helm:
+
+  - The installation chart for the Smart Agent is compatible with Helm version 2 or Helm version 3.
+  - To learn more, see the [Helm site](https://helm.sh/).
+
+* Tiller installed on all of your Kubernetes hosts. Because of
+  the way that the installation chart works, you need Tiller even if you're using
+  Helm version 3.
+* SignalFx realm for your organization. See [Realms](../../../_sidebars-and-includes/smart-agent-realm-note.html).
+* A SignalFx access token. See [Smart Agent Access Token](../../../_sidebars-and-includes/smart-agent-access-token.html).
+
+
+## Configure Helm for the Smart Agent
+
+The SignalFx Smart Agent chart for Helm comes with a values.yaml configuration
+file that contains useful default values. If you want to modify these values,
+override them by creating your own values file. Follow these steps:
+
+1. Download the default [values.yaml](https://github.com/signalfx/signalfx-agent/blob/master/deployments/k8s/helm/signalfx-agent/values.yaml) file.
+2. Rename the file. For example, rename it to myValues.yaml.
+3. Edit the file to update the values with your own choices, then save the file.
+
+When you install the Smart Agent using Helm, add the `-f <values_yaml_file>` parameter to the install command.
+For example, add the `-f myValues.yaml` parameter.
+
+## Install with Helm
+
+1. Remove collector services such as `collectd`.
+2. Remove third-party instrumentation and agent software.
+
+> Don't use automatic instrumentation or instrumentation agents from
+> other vendors when you're using SignalFx instrumentation. The results
+> are unpredictable, and your instrumentation might break and your
+> application might crash.
+
+3. Add the SignalFx Helm chart repository to Helm by enter the following command:
+
+   ```
+   helm repo add signalfx https://dl.signalfx.com/helm-repo
+   ```
+
+4. Check that the repository is up-to-date by entering the following command:
+
+   ```
+   helm repo update
+   ```
+
+5. Get the following required values:
+
+| Name             | Example       | Meaning                                                          |
+|:-----------------|:--------------|:-----------------------------------------------------------------|
+| `<access_token>` | 'abcd1234zzz' | Required. Access token. See [Prerequisites](#prerequisites). |
+| `<cluster_name>` | 'myCluster'   | Required.Name of the cluster to monitor                      |
```

Comment:
cluster should not be required anywhere

Reply:
I'll double-check. It was listed in the instructions at the time I created the document.

Reply:
Found a reference to it as a requirement. See [Smart Agent Quick Install for Kubernetes using Helm](https://docs.signalfx.com/en/latest/integrations/kubernetes/k8s-quick-install.html#smart-agent-quick-install-for-kubernetes-using-helm). Please advise.